policy/modules/services/dbus.te

```diff
@@ -172,6 +172,8 @@ ifdef(`init_systemd', `
 
 	# Recent versions of dbus are started as Type=notify
 	init_write_runtime_socket(system_dbusd_t)
+
+	allow system_dbusd_t init_t:unix_stream_socket connectto;
```

Comment:
Idem ditto: should (probably) be addressed as part of auth_use_nsswitch()
https://github.com/SELinuxProject/refpolicy/pull/269

Reply:
Good to know, that can be dropped from the PR then. Thanks!